lib/pbio/src/os.c

```diff
@@ -47,15 +47,23 @@ void pbio_os_request_poll(void) {
 static pbio_os_process_t *process_list = NULL;
 
 /**
- * Starts a process.
+ * Placeholder thread that does nothing.
  *
- * NB: Processes can be started only once. They cannot be restarted.
+ * @param [in] state The protothread state.
+ * @param [in] context The context.
+ */
+pbio_error_t pbio_port_process_none_thread(pbio_os_state_t *state, void *context) {
```

Comment:
I don't understand this function name. But if it is just temporary, I guess it doesn't matter.